### PR TITLE
[account] Generic Improvements on Invoicing

### PR DESCRIPTION
**Pull request by hiralvbhavsar**
Description of the issue/feature this PR addresses:

Task: https://www.odoo.com/web?#id=58392&view_type=form&model=project.task&action=333&active_id=131&menu_id=4720
Pad: https://pad.odoo.com/p/r.ae45a02aedc306d377d8164e4eb45c7a

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by rgo-odoo** [comment]
LGTM

File: addons/account/models/account_journal_dashboard.py [hunk 1]
```diff
@@ -413,6 +413,38 @@ def create_bank_statement(self):
         })
         return action
 
+    @api.multi
+    def create_customer_payment(self):
+        """return action to create a customer payment"""
+        action = self.env.ref('account.action_account_payments').read()[0]
+        action.update({
+            'views': [[False, 'form']],
+            'context': "{'default_payment_type': 'inbound', 'default_partner_type': 'customer', 'default_journal_id': " + str(self.id) + "}",
```
Comment (fmdl) [inline addons/account/models/account_journal_dashboard.py]: you can write the context without "" :

'context': {'default_payment_type': 'inbound', 'default_partner_type': 'customer', 'default_journal_id': self.id)},

File: addons/account/models/account_journal_dashboard.py [hunk 2]
```diff
@@ -413,6 +413,38 @@ def create_bank_statement(self):
...
+        })
+        return action
+
+    @api.multi
+    def create_supplier_payment(self):
+        """return action to create a supplier payment"""
+        action = self.env.ref('account.action_account_payments_payable').read()[0]
+        action.update({
+            'views': [[False, 'form']],
+            'context': "{'default_payment_type': 'outbound', 'default_partner_type': 'supplier', 'default_journal_id': " + str(self.id) + "}",
```
Comment (fmdl) [inline addons/account/models/account_journal_dashboard.py]: you can write the context without "" :

**Comment by qdp-odoo** [comment]
landed in master, for saas-11.4, at revision https://github.com/odoo/odoo/commit/42b7d7489a1dc3eaee204007a8286f063e173356

**Pull request closed without merging**